article.html

```diff
@@ -16,102 +16,113 @@
+
+                </div> -->
+                <form action="">
+                    <!-- <button type="submit" id="like_button"></button> -->
+                    <i id="like_icon"></i>
+                </form>
+                <p class="no_likes" id="no_likes"></p>
             </div>
             <div class="comments">
                 <i class="fa-regular fa-message"></i>
-                <p class="no_comments">2</p>
+                <p class="no_comments" id="no_comments"></p>
+            </div>
+            <div class="copy">
+                <i class="fa-regular fa-clone" id="copy_url"></i>
+                <p id="copied"></p>
             </div>
-            <i class="fa-regular fa-clone"></i>
         </div>
         <div class="add_comment">
-            <img src="images/author.jpg" alt="author_image">
-            <input type="text" placeholder="Add comment">
+            <div class="icon_container">
+                <i class="fa-solid fa-user"></i>
+            </div>
+            <form action="" id="comment_form">
+                <input type="text" placeholder="Add comment" oninput="showButton()" id="comment_content" name="comment_content">
+                <button id="send_comment">Send</button>
+            </form>
         </div>
-        <div class="comments_list">
-            <div class="single_comment">
+        <div class="comments_list" id="comments_list">
+            <!-- <div class="single_comment">
                 <div class="icon_container">
                     <i class="fa-solid fa-user"></i>
                 </div>
                 <div class="text">
                     <div class="comment_header">
-                        <p class="name">John Doe</p>
-                        <p class="date">1 Jan 2023</p>
+                        <p class="name" id="commenter_name"></p>
+                        <p class="date" id="commenting_date"></p>
                     </div>
-                    <p class="comment">Donec tincidunt sapien ante, id aliquam sapien fermentum non. Suspendisse blandit ornare dui vel pretium. Cras tristique magna nisl, aliquam lacinia metus pharetra eu. Fusce efficitur tempor leo, quis hendrerit odio sodales eu.</p>
-                </div>
-            </div>
-            <div class="single_comment">
-                <div class="icon_container">
-                    <i class="fa-solid fa-user"></i>
+                    <p class="comment" id="comment_body"></p>
                 </div>
-                <div class="text">
-                    <div class="comment_header">
-                        <p class="name">Jane Doe</p>
-                        <p class="date">1 Jan 2023</p>
-                    </div>
-                    <p class="comment">Donec tincidunt sapien ante, id aliquam sapien fermentum non. Suspendisse blandit ornare dui vel pretium. Cras tristique magna nisl, aliquam lacinia metus pharetra eu. Fusce efficitur tempor leo, quis hendrerit odio sodales eu.</p>
+            </div> -->
```

Comment:
Am not sure why you commented these big lines of codes I would suggest removing the if they're not doing anything